hexmap.py

```diff
@@ -93,28 +113,52 @@ def base_row(base_row_string):
     return row_string
 
 
+
 def starmap_string(starmap):
     global row
     # stellagama.clear_screen()
     star_string = f" UNIVERSAL OS v.21.1\n\n A U T O N O M O U S    R E G I O N\n\n {base_row('  _____       ')}\n"
-
+    
+    # Convert starmap to preliminary format.
+    # Format is a 3d array, Row, Column, Line
+    printmap = [[[] for i in range(9)] for j in range(11)]
+    print("Created Array")
+    for row in range(0,11):
+        for column in range(1,9):
+            if starmap[column][row] is not None:
+                printmap[row][column].append(f"  {starmap[column][row].starport} {starmap[column][row].gas_giants}")
+                printmap[row][column].append(f"{starmap[column][row].naval_base()}  {starmap[column][row].worldtype()}   ")
+                printmap[row][column].append(f"{name_converter(starmap[column][row].name)}")
+                printmap[row][column].append(f"{starmap[column][row].scout_base()}{hex_number(column, row, starmap[column][row].worldtype)}")

```

Comment:
I think I like putting this into `World2` more, but here's another way of doing this:

```python
def print_star_cell(cell, star_cell):
    cell.append(f"  {starmap[column][row].starport} {starmap[column][row].gas_giants}")
    cell.append(f"{starmap[column][row].naval_base()}  {starmap[column][row].worldtype()}   ")
    cell.append(f"{name_converter(starmap[column][row].name)}")
    cell.append(f"{starmap[column][row].scout_base()}{hex_number(column, row, starmap[column][row].worldtype)}")
```

```suggestion
                print_cell(printmap[row][column], starmap[column][row])
```